kvmagent/kvmagent/plugins/qga_zwatch.py

```
@@ -211,6 +215,14 @@ def zwatch_qga_monitor_vm(self, uuid, qga):
                 time.sleep(self.push_interval_time)
             except Exception as e:
                 logger.warn('vm[%s] end monitor with qga due to [%s]' % (uuid, str(e)))
+                # error in restarting zwatch agent service, set qgaZwatch to empty
+                if zwatch_restart_cmd and zwatch_vm_info_path and zwatch_restart_cmd in str(e):
+                    try:
+                        qga.guest_file_write(zwatch_vm_info_path, '')
+                    except Exception as fe:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)